site/resources.md

```diff
@@ -73,9 +73,9 @@ Resources
 
   _add description_
 
-* MetPy (basic calculations)
+* [MetPy (basic calculations)](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/metpy_basics.ipynb)
```

Comment:
Please bold the title, remove the new line between the title and description, and add two blank spaces to the end of the title, after the link.  (see the python-awips entries in this markdown for reference if needed)